library/data/tags/src/main/java/org/quiltmc/qsl/tag/api/QuiltTagBuilder.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.tag.api;
+
+import net.minecraft.tag.Tag;
+
+/**
+ * Interface implemented by {@link net.minecraft.tag.Tag.Builder} instances when QSL is present.
+ *
+ * @see org.quiltmc.qsl.tag.api
+ */
+public interface QuiltTagBuilder {
+	/**
+	 * Clears the contained entries and mark the tag as replaced.
+	 */
+	void clearEntries();
```

Comment:
Would returning `QuiltTagBuilder` here be better? Creating a local for a simple builder call (or using `Util.make` for it) feels a bit odd to me.

Reply:
Good call, will do that.